modules/integration/test-ui-sp-editor/src/main/java/org/wso2/sp/selenium/components/actionObjects/RootActionObject.java

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.sp.selenium.components.actionObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class RootActionObject {
```

Comment:
Having the "Object" suffix in a class name is very weird. Java is a OOP language, so every class is a template of an Object. So no point of having the word "Object" in the class name. For example, we don't say `public class CatObject`, we simply say `public class Cat`.

Suggesting to change the class name to `RootAction` and the package name to `org.wso2.sp.selenium.components.action`. This applies to other places as well.

Reply:
We had the naming issue but Dharshana suggested to have class names like the above